code/obj/item/clothing/gloves.dm

```diff
@@ -479,6 +479,73 @@ ABSTRACT_TYPE(/obj/item/clothing/gloves)
 		..()
 		BLOCK_SETUP(BLOCK_ROPE)
 
+/obj/item/clothing/gloves/bladed
+	desc = "Transparent gloves make it look like the wearer isn't wearing gloves at all. There's a small gap on the back of each glove."
+	name = "transparent gloves"
+	icon_state = "transparent"
+	item_state = "transparent"
+	material_prints = "transparent high-quality synthetic fibers"
+	activeweapon = FALSE
+	hit_type = DAMAGE_CUT
+	force = 0
+	stamina_damage = 0
+	stamina_cost = 0
+	attack_verbs = "slashes"
+	hitsound = 'sound/impact_sounds/Blade_Small_Bloody.ogg'
+	var/deployed = FALSE
+
+	nodescripition = TRUE
+
+	special_attack(mob/living/target, mob/living/user)
+		if(check_target_immunity( target ))
+			return 0
+		logTheThing(LOG_COMBAT, user, "slashes [constructTarget(target,"combat")] with hand blades at [log_loc(user)].")
+		var/obj/item/affecting = target.get_affecting(user)
+		var/datum/attackResults/msgs = user.calculate_melee_attack(target, affecting, 16, 16, 0)
+		user.attack_effects(target, affecting)
+		var/action = pick("stab", "slashe")
+		msgs.base_attack_message = "<b><span class='alert'>[user] [action]s [target] with their hand blades!</span></b>"
+		msgs.played_sound = 'sound/impact_sounds/Blade_Small_Bloody.ogg'
+		msgs.damage_type = DAMAGE_CUT
+		msgs.flush(SUPPRESS_LOGS)
+		user.lastattacked = target
+
+	proc/sheathe_blades_toggle(mob/living/user)
+		playsound(src.loc, 'sound/effects/sword_unsheath1.ogg', 50, 1)
+
+		if(deployed)
+			deployed = FALSE
+			force = 0
+			stamina_damage = 0
+			stamina_cost = 0
```

Comment:
to go with the above - use `initial(foo)` to reset these to defaults instead of zeroing them out